### ISSUE TITLE
All attribution links should open in new browser tab - including Leaflet

### ISSUE THREAD

**Issue by gjlawran** [bcgov/ckanext-bcgov#124]
Leaflet attribution hyperlink used in Data Explorer web map should open in a new tab - rather than within the current iFrame. This is an attribution link that is built into default Leaflet -not added through configuration -  so a modification to either Leaflet or recline.js may be required to facilitate.

**Comment by jrods** [comment]
@gjlawran can you provide an example resource that does this?

**Comment by gjlawran** [comment]
Visit this resource https://catalogue.data.gov.bc.ca/dataset/bc-public-libraries-systems-branches-and-locations/resource/b6a28bfb-580b-4662-9f00-f7189d52fbe6
Select the Map tab in the Data Explorer
Select the Leaflet attribution hyperlink.

**Comment by jrods** [comment]
The Leaflet attribution link is added by the library itself. The change would have to be done to the vendor file, leaflet-src.js the reclineview ext in ckan.

**Comment by kfishwick** [comment]
This is not part of the core behaviour and is difficult to change (see above details from @jrods).  I would recommend revisiting this requirement & deciding if it is really necessary.

**Comment by Mbrownshoes** [comment]
Agreed, we can delay for now.

**Comment by jeff-at-h3** [comment]
is this still a valid issue? needs to be determined before we can estimate

**Comment by Mbrownshoes** [comment]
@jeff-at-h3  Yes, this is still a valid issue. leaflet attribution doesn't open in new window:
https://catalogue.data.gov.bc.ca/dataset/bc-public-libraries-systems-branches-and-locations-2016/resource/b6a28bfb-580b-4662-9f00-f7189d52fbe6?view_id=3353cdb5-ef17-4f83-9bf1-c67bfdce3912
